### DISCUSSION TITLE
Teslamate on 64bit Raspberry Pi OS?

### DISCUSSION THREAD

**Discussion by martinh2011** [teslamate-org/teslamate#2590] [Q&A]
Would Teslamate run on 64bit Raspberry Pi OS? The docker image seem to be available for arm64, too. But before I try To update my Raspi 4 I would like to know if anybody already does run it successfully?

**Comment by rc-16** [comment]
I run Teslamate on a raspberry 4b with 4G RAM. It is dedicated to Teslamate and I've not done anything fancy - just followed the documentation for the install. I have recently updated to the latest Raspberry OS - Bullseye. Again no pain. 

Thank you to the wonderful team that have created and support this amazing tool.

**Reply by martinh2011** [reply]
Thanks. I'm using Teslamate on the same hardware. However, Raspberry PI OS in February released a 64bit version of the OS. I'm still running the 32bit version and I'm wondering if anybody is already using the 64bit version with Teslamate.

**Reply by cleanev** [reply]
I have been running on 64 bit since Raspbian 64-bit became GA.

Linux teslamate 5.15.32-v8+ #1538 SMP PREEMPT Thu Mar 31 19:40:39 BST 2022 aarch64

**Comment by ktsenvolt** [comment]
![bullseye os on raspi](https://user-images.githubusercontent.com/80360674/162442238-e4ae4f4f-17e7-4aa8-b80a-9722918b1354.jpg)
I am running on 32bit version as well.(Advanced installation with Traefik, Let's Encrypt & HTTP Basic Auth)
My next project will be working  on 64bit.This may take sometime.
